common/Environments/Helpers/ComputeSystemHelpers.cs

```diff
@@ -102,4 +102,28 @@ public static CardStateColor GetColorBasedOnState(ComputeSystemState state)
             _ => CardStateColor.Caution,
         };
     }
+
+    public static (bool, string?, string?) UpdateCallToActionText(int providerCount, bool isCreationPage = false)

```

Comment:
You can add variable names here so it's easier to understand which is which (bool navigateToExtensionsLibrary, string? callToActionText, string? callToActionHyperLinkText)

Reply:
updated and made it into a class now. Thanks